tests/components/homekit/test_accessories.py

```diff
@@ -89,13 +90,17 @@ def test_home_accessory(self):
         hass.states.set('homekit.accessory', 'off')
         hass.block_till_done()
 
+        acc = HomeAccessory(hass, 'Home Accessory', 'test_model.accessory', 2,
```

Comment:
Just a minor thing. If we use `test_model.acc` instead, it should fit on one line.

Reply:
Done.